examples/hello-world-http/registerUnregisterHTTPExample.js

```diff
@@ -0,0 +1,21 @@
+const {HttpService, Reshuffle} = require('../../');
+
+const app = new Reshuffle();
+const service = new HttpService();
+
+app.register(service);
+
+// Check 2 starts
+app.start();
+app.start();
```

Comment:
why do we need to two app.start()?

Reply:
oh it was part of my test to make sure the app doesn't crash and doesn't try to create a second express web server. This file will go once we have sufficient test coverage